### PR TITLE
optimization verion command

### PR DESCRIPTION
**Pull request by lengrongfu**
Fixes: https://github.com/volcano-sh/volcano/issues/3579

Test result:
image

### CODE REVIEW

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

File: cmd/scheduler/app/server.go [hunk 1]
```diff
@@ -56,10 +55,6 @@ import (
 
 // Run the volcano scheduler.
 func Run(opt *options.ServerOption) error {
-	if opt.PrintVersion {
-		version.PrintVersionAndExit()
```
Comment (googs1025) [inline cmd/scheduler/app/server.go:60]: I have a question: Why do we need to move it to main.go? Can we just return here?

File: cmd/webhook-manager/main.go [hunk 2]
```diff
@@ -29,7 +29,8 @@ import (
 
 	"volcano.sh/volcano/cmd/webhook-manager/app"
 	"volcano.sh/volcano/cmd/webhook-manager/app/options"
-
+	"volcano.sh/volcano/pkg/version"
```
Comment (googs1025) [inline cmd/webhook-manager/main.go:32]: nit: import lint need to be fixed

File: cmd/scheduler/app/server.go (hunk 1 above)
Comment (lengrongfu) [reply cmd/scheduler/app/server.go:60]: Maybe it's in this location running error, but i only need know version info.
https://github.com/volcano-sh/volcano/blob/fb068bde7a0cf80392b225517dd3dd522524091a/cmd/scheduler/main.go#L65-L67

File: cmd/webhook-manager/main.go (hunk 2 above)
Comment (lengrongfu) [reply cmd/webhook-manager/main.go:32]: done

File: cmd/scheduler/app/server.go (hunk 1 above)
Comment (lengrongfu) [reply cmd/scheduler/app/server.go:60]: Do you have any idea? @googs1025

File: cmd/scheduler/app/server.go [hunk 3]
```diff
@@ -56,10 +55,6 @@ import (
 
 // Run the volcano scheduler.
 func Run(opt *options.ServerOption) error {
-	if opt.PrintVersion {
```
Comment (Monokaix) [inline cmd/scheduler/app/server.go:59]: Hi, welcome! A little question here, when doing secondary development based on volcano and we need to add additional `opt.PrintVersion` when calling `app.Run`, which is not very friendly to developers.

Comment (lengrongfu) [reply cmd/scheduler/app/server.go:59]: So do we add a `PrintVersion` in main.go and also add a `PrintVersion` in Run?

**Pull request merged**